src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import { Person } from '../../types';
+import { PersonInfo } from '../PersonInfo/PersonInfo';
+import { useSearchParams } from 'react-router-dom';
+import { ColumnFilter } from '../ColumnFilter/ColumnFilter';
+import { clearPeopleList } from '../../utils/clearPeopleList';
+
+type Props = {
+  people: Person[];
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+
+  const centuries = searchParams.getAll('centuries');
+  const sex = searchParams.get('sex');
+  const query = searchParams.get('query');
+  const sort = searchParams.get('sort');
+  const order = searchParams.get('order');
+
+  const visiblePeople = clearPeopleList(
+    people,
+    centuries,
+    sex,
+    query,
+    sort,
+    order,
+  );
+
+  if (!visiblePeople.length) {
+    return <p>There are no people matching the current search criteria</p>;
```

Comment:
Consider adding a `data-cy` attribute to the message for better testing and consistency with other elements.